static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,177 @@
+# Data Registry
+
+In the [Versioning Tutorial](/doc/tutorials/versioning) we use two ZIP files
+containing parts of a dataset with labeled images of cat and dogs. For
+simplicity, these compressed archives are downloaded with `dvc get` from our
+[dataset registry](https://github.com/iterative/dataset-registry), a <abbr>DVC
+project</abbr> hosted on GitHub.
+
+In this document we'll explain the idea behind **shared data registries**, how
+to easily create one with DVC, and the best ways to version datasets based on
+the same example above (without ZIP files!)
+
+## Concept
```

Comment:
it feels like this can serve as an even better intro as the previous two paragraphs. Or simplify them and make about dataset registry, nor versioning example and its specifics. I really like the Concept intro, motivation, etc.

Reply:
Hmm you're totally right, I'll use that as intro and remove the "Concept" header which was just a placeholder, really...

Reply:
OK done! And merged the old intro with the example intro. I still need to compress all the 2nd half as we discussed on Slack...